backend_api/README.md

```diff
@@ -16,87 +16,72 @@ Create your App Engine application by typing:
 
 You will have to select a region. Choose one that is close to your location.
 
-Next, set an environment variable for your project ID:
-
-    export PROJECT_ID=$(gcloud config get-value core/project)
-
 ## Enable APIs and create a service account
```

Comment:
API enabling will actually be run in the project creation script, so we can take it out (all APIs will be enabled on project creation so that they don't need to have the permission to enable APIs)